WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/SpeakerScreen.kt

```diff
@@ -142,14 +107,45 @@ fun SpeakerScreen(
     }
 }
 
-class AppStatePreviewProvider : CollectionPreviewParameterProvider<AppState>(
+private fun createControlDashboardUiState(
```

Comment:
This should be remembered, otherwise it'll be created on every recomposition

Reply:
Updated!

Reply:
Sorry, I should've explained myself better. I meant remembering the `derivedStateOf` function. Something like

```
= remember {
    derivedStateOf { ... }
}
```

I believe that's more efficient than doing `remember(x,y)` but I'm not 100% sure, maybe we need to check with eng to see what's preferable?

Reply:
Also, I've seen that we have a usage of `derivedStateOf` in Jetnews which isn't remembered. We might need to fix that in the same way. Thanks!

Reply:
Per chats about `derivedStateOf` and `remember`, I'm removing it entirely from these usages.

The computations are simply enough that there's no need for overcomplicating them, and using `derivedStateOf` properly is surprising subtle.

I kept the extracted function, but it's now a plain old Kotlin function that computes the ui state.